keyboards/apple_m5120/iso/config.h

```diff
@@ -0,0 +1,39 @@
+ /* Copyright 2020 Damien Broqua
+  *
+  * This program is free software: you can redistribute it and/or modify
+  * it under the terms of the GNU General Public License as published by
+  * the Free Software Foundation, either version 2 of the License, or
+  * (at your option) any later version.
+  *
+  * This program is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x991D
+#define DEVICE_VER 0x0001
+#define MANUFACTURER Apple
```

Comment:
May want to change this. You are not Apple.

Reply:
I'm not Apple but the original Keyboard is from Apple and I use his PCB + Teensy. I'm not the creator of the Keyboard.

I need to set what ?

- my name? I'm not the PCB designer
- Apple? But the controler is a Teensy and the wiring is from my mind
- Teensy? But the PCB is from Apple/Alps
- My name + Apple/Alps + Teensy ?

I have no idea of what is the best.

Reply:
I guess this is technically a handwired board then, and so it should go in `handwired/` and have your name.

Reply:
It's a mix, original PCB with Teensy on his J1 connector (halfway between handwired and usb converter).

https://pics.darkou.fr/H3vPfmUg.png